app/views/provider/admin/api_docs/show.html.erb

```diff
@@ -1,8 +1,4 @@
-<%= stylesheet_link_tag 'active-docs/application.css' %>
-<%= javascript_tag do %>
-  window.enableApiDocsProxy = <%= Rails.configuration.three_scale.active_docs_proxy_disabled.blank? %>;
-<% end %>
-<%= javascript_include_tag 'active-docs/application.js' %>
+<%= javascript_pack_tag 'provider_active_docs' %>
```

Comment:
What's that proxy for?

Reply:
ApiDocs proxy is a feature available only in SaaS. It makes the requests that are made from SwaggerUI pass through a proxy that has CORS enabled. This is for APIs that do not have CORS enabled - and the feature allows these requests to work OK from the admin or developer portal.
The proxy is based on nginx and its source code is here https://github.com/3scale/cors-proxy

Reply:
We don't need it here, because:
1. I think 3scale backend (for Service Mgmt API) has CORS already enabled
2. Requests to system API are using the same domain, so CORS does not apply here